actors/builtin/multisig/multisig_actor.go

```diff
@@ -163,7 +162,7 @@ func (a *MultiSigActor) AddSigner(rt vmr.Runtime, params *AddSigner) *vmr.EmptyR
 	var st MultiSigActorState
 	rt.State().Transaction(&st, func() interface{} {
 		if st.isSigner(params.Signer) {
-			rt.AbortStateMsg("party is already a signer")
+			rt.Abort(exitcode.ErrIllegalState, "party is already a signer")
```

Comment:
ErrForbidden?

Reply:
I took forbidden to mean a forbidden arg as in one that could not be valid (e.g. neg int or something) not one that happens not to be. I may be wrong. Which is it?

Reply:
ah fair

Reply:
but it's not you can't make him a signed, it's he already is...?